src/main/java/JavaQuestionTwo/addUser.java

```diff
@@ -0,0 +1,86 @@
+package JavaQuestionTwo;
+import java.util.*;
+import java.io.*;
+
+
+public class addUser {
+
+	public ArrayList<user> users ;
+	public user user1;
+	public HashSet<Integer> rollSet= new HashSet<>();
+	Scanner sc = new Scanner(System.in);
+
+	public addUser() {
+		displayUser disp = new displayUser();
+		this.users=disp.readFromFile();
+
+		for(user u1:users)
+			rollSet.add(u1.getRollno());
+	}
+
+	public ArrayList<user> getUsers() {
+		return users;
+	}
+	public void setUsers(ArrayList<user> user) {
+		users = user;
+	}
+
+	public void AddUser() {
+
+		String name , address;
+		int age,rollno;
+		ArrayList<String> subj = new ArrayList<String>();
```

Comment:
```suggestion
		List<String> subj = new ArrayList<String>();
```
program to an interface when dealing with collections i.e map, list etc.